source/configure/web-server-configuration-settings.rst

```diff
@@ -470,13 +470,11 @@ Trusted proxy IP header
 +--------------------------------------------------------+------------------------------------------------------------------------------+
 | **Notes**:                                                                                                                            |
 |                                                                                                                                       |
-| - From Mattermost v5.12, new deployments set this value to ``[]``, meaning that no header will be trusted. Prior to v5.12, the        |
```

Comment:
Please revert this line removal and start the sentence with "The default value of ``[]`` means that no header will be trusted."